mvrx/src/main/kotlin/com/airbnb/mvrx/PersistState.kt

```diff
@@ -31,30 +28,31 @@ annotation class PersistState
  * Iterates through all member properties annotated with [PersistState] and parcels them into a bundle that can be
  * saved with savedInstanceState.
  */
-internal fun <T : Any> T.persistState(assertCollectionPersistability: Boolean = false): Bundle {
-    val klass = this::class
-    val persistStateArgs = klass.primaryConstructor
-            ?.parameters
-            ?.filter { it.annotations.any { it.annotationClass == PersistState::class } }
-    if (persistStateArgs?.isEmpty() != false) {
-        return Bundle()
-    }
+internal fun <T : MvRxState> T.persistState(validation: Boolean = false): Bundle {
+    val jvmClass = this::class.java
+    // Find the first constructor annotated with @PersistState or return.
+    val constructor = jvmClass.constructors.firstOrNull { it.parameterAnnotations.any { it.any { it is PersistState } } } ?: return Bundle()
```

Comment:
The comment here says it's looking for constructors annotated with `@PersistState`, but the implementation appears to be looking for the first constructor with any _parameters_ with that annotation?